apps/web/layouts/PageLayout/AuthLayout/Sidebar.tsx

```diff
@@ -0,0 +1,103 @@
+import Image from 'next/image'
+import Link from 'next/link'
+
+import {
+  ArrowLeftRightIcon,
+  DatabaseIcon,
+  DocsIcon,
+  ExportIcon,
+  ProfileIcon,
+  SupportIcon,
+} from '../../../components/icons'
+
+import type {NavLinkProps} from './NavLink'
+import {NavLink} from './NavLink'
+
+const mainNavigation: NavLinkProps[] = [
+  {
+    name: 'Connections',
+    href: '/pipelines',
+    icon: ArrowLeftRightIcon,
+  },
+  {name: 'Export Data', href: '/export', icon: ExportIcon},
+  {
+    name: 'API Access',
+    href: '/api-access',
+    icon: DatabaseIcon,
+  },
+]
+
+const helpNavigation: NavLinkProps[] = [
+  {
+    name: 'Support',
+    href: '/support',
+    icon: SupportIcon,
+  },
+  {name: 'Documentation', href: '/docs', icon: DocsIcon},
```

Comment:
This will 404 for now.

Reply:
This should point to https://docs.venice.is (new window)

Reply:
🙏 7e4788b